docs/wrappers/osnadmin_channel_preamble.md

```diff
@@ -0,0 +1,13 @@
+# osnadmin channel
+
+The `osnadmin channel` command allows administrators to perform channel-related
+operations on an orderer, such as joining a channel, listing the channels an
+orderer has joined, and removing a channel.
```

Comment:
Would it be appropriate to mention here that the join and remove commands cannot be used if the ordering node is part of a system channel?

Reply:
I can certainly add that if you think it'd be beneficial.

Reply:
Updated